pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1281,15 +1280,13 @@ func podToMemberPod(pod *v1.Pod, includeIP, includePodRef bool) *controlplane.Gr
 		}
 	}
 	if includeIP {
-		memberPod.IP = ipStrToIPAddress(pod.Status.PodIP)
+		memberPod.IPs = []controlplane.IPAddress{ipStrToIPAddress(pod.Status.PodIP)}
 	}
-	if includePodRef {
-		podRef := controlplane.PodReference{
-			Name:      pod.Name,
-			Namespace: pod.Namespace,
-		}
-		memberPod.Pod = &podRef
+	podRef := controlplane.PodReference{
```

Comment:
From code, I think both versions of AddressGroup will have PodRef set, right?
Since @jianjuns proposed to remove them from AddressGroup to reduce size, @jianjuns for his attention.

Reply:
I think we should keep that information unless there is an actual performance issue reported. This information enables us to build more advanced tools for policy analysis IMO, and makes dumping the groups with antctl more meaningful.

Reply:
It depends on the controller version. The upgraded networkpolicy controller will set PodRef for AddressGroups while the old version will not. The PodRef is for when the old versioned agent watching v1beta1 resource from the updated controlplane resource to work as expected

Reply:
> From code, I think both versions of AddressGroup will have PodRef set, right?
> Since @jianjuns proposed to remove them from AddressGroup to reduce size, @jianjuns for his attention.

It depends on the controller version. The upgraded networkpolicy controller will set PodRef for AddressGroups while the old version will not. The PodRef is for when the old versioned agent watching v1beta1 resource from the updated controlplane resource to work as expected